jac-uk/admin
# WIP #1345 Configure application form content

## What's included?
First iteration of Staged Applications epic.
Here we are enabling the application form to be configured.
The work has involved re-organising our code with the following changes:
 - New folder structure which groups together related code
 - New routing
 - New sub navigation and side navigation
 - Create an exercise journey should now 'work' better than it did

## Who should test?
✅ Product owner
✅ Developers

## How to test?
As there have been lots of code changes it would be good to test the whole Exercise area of admin including:
 - Create an exercise, with a 'journey'. User should expect to be navigated through the selected forms for configuring the exercise
 - Create an exercise without a 'journey'
 - Navigate around and configure a new exercise
 - Navigate around and use an existing exercise including tasks, stages, reports etc.
 - Use the new 'Application form' area to configure which data is requested from the candidate when they apply for the vacancy

## Risk - how likely is this to impact other areas?
🔴 High risk - this includes a lot of changes to shared code

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
